### PR TITLE
[fix](nereids)fix bug that query infomation_schema.rowsets fe send fragment to one of muilti be.

### PR DESCRIPTION
**Pull request by hubgeter**
Fixed the bug of incomplete query results when querying information_schema.rowsets in the case of multiple BEs.

The reason is that the schema scanner sends the scan fragment to one of multiple bes, and be queries the information of fe through rpc. Since the rowsets information requires information about all BEs, the scan fragment needs to be sent to all BEs.

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hubgeter** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java [hunk 1]
```diff
@@ -725,13 +726,21 @@ public PlanFragment visitPhysicalSchemaScan(PhysicalSchemaScan schemaScan, PlanT
         Table table = schemaScan.getTable();
         List<Slot> slots = ImmutableList.copyOf(schemaScan.getOutput());
         TupleDescriptor tupleDescriptor = generateTupleDesc(slots, table, context);
-        SchemaScanNode scanNode = new SchemaScanNode(schemaScan.translatePlanNodeId(), tupleDescriptor);
+        SchemaScanNode scanNode = null;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java:729]: please add comment to explain this logic

**Review by morrySnow — COMMENTED** [review]
please add description about this PR

**Comment by hubgeter** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by hubgeter** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by morningman — APPROVED** [review]

**Pull request merged**